test/Fortran/source/test_RTS_RepeatsFloatsToDouble.f90

```diff
@@ -43,21 +43,20 @@ SUBROUTINE test_RTS_RepeatsFloatsToDouble(ifltab1, messageUnit, status)
 !
       nvals = 100
       j = 1
-      do 120 i=1, nvals
+      do i=1, nvals
        data1(i) = FLOAT(i)
        data3(i) = data1(i) 
-       !do 110 j=1,2
         quality1(1,i) = (i*100) + j
```

Comment:
The loop was commented out, so j always = 1 (as per line 45)

Reply:
I think j=1 the first iteration, and j=3 for all others, because of the nested do loop.  I guess it doesn't matter, the test is writing and reading, and checking for consistency.